docs/get-started/get-started.md

```diff
@@ -239,4 +242,172 @@ CompositionLocalProvider(LocalGravatarTheme provides object : GravatarTheme {
 }) {
     LargeProfileSummary(profile = userProfile)
 }
+```
+
+## Quick Editor
+
+The `:gravatar-quickeditor` module provides a fully featured component that allows the user to modify their avatar without leaving your app. 
+
+To do that the QuickEditor needs an authorization token to perform requests on behalf of the user. There are two ways for that:
+
+### 1. Obtain the token yourself and provide it to the Quick Editor
+
+Quick Editor can be launched with the provided token. To obtain it, you have to follow the [OAuth docs](https://docs.gravatar.com/?page_id=2555&preview=true) and implement the OAuth flow within your app.
```

Comment:
💡 I wonder if we should position this method first instead of "Leaving the OAuth flow to Quick Editor." As the other method is more straightforward and takes care of saving the token, I would switch the order.

Reply:
Sure - from the SDK integration point of view, the simpler way is if you provide the token, but only for the apps that have it already. I'm okay with reordering.

Reply:
Done